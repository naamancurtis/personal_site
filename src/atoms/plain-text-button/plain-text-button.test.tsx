import React from 'react';
import { shallow } from 'enzyme';
import {
  PressButton,
  CodeButton,
  LiveButton,
} from './plain-text-button.component';

describe('<PressButton />', () => {
  it('expect to render component', () => {
    const wrapper = shallow(<PressButton url="test" />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('<CodeButton />', () => {
  it('expect to render component', () => {
    const wrapper = shallow(<CodeButton url="test" />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('<LiveButton />', () => {
  it('expect to render component', () => {
    const wrapper = shallow(<LiveButton url="test" />);
    expect(wrapper).toMatchSnapshot();
  });
});
