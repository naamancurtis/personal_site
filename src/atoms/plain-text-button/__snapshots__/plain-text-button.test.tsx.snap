// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CodeButton /> expect to render component 1`] = `
<BaseButton
  icon={
    Array [
      "fas",
      "code",
    ]
  }
  text="Code"
  url="test"
/>
`;

exports[`<LiveButton /> expect to render component 1`] = `
<BaseButton
  icon={
    Array [
      "fas",
      "desktop",
    ]
  }
  text="Live"
  url="test"
/>
`;

exports[`<PressButton /> expect to render component 1`] = `
<BaseButton
  icon={
    Array [
      "far",
      "newspaper",
    ]
  }
  text="Article"
  url="test"
/>
`;
